test/test_pytest_fixtures.py

```diff
@@ -51,3 +50,67 @@ def test_model_base(model_base):
     result.stdout.fnmatch_lines(
         ["*NotImplementedError*"]
     )
+
+
+class TestCreateEngineKwargs(object):
+
+    def test_default_create_engine_kwargs(self, create_engine_kwargs):
+        assert create_engine_kwargs == {}
```

Comment:
I made the fixture return an empty dictionary by default, which will not make any effect in the current code and will be backwards compatible for all the services that currently use it.

If we need to provide DB engine options in our services then we can always override this fixture, as I did in the README examples. What do you think?